src/App.js

```diff
@@ -1,5 +1,44 @@
+import { OUTPUT_MESSAGE_WINNER } from './lib/constants.js';
+import { print } from './lib/utils.js';
+import Input from './Input.js';
+import Race from './Race.js';
+
 class App {
-  async run() {}
+  #carArray;
+
+  #tryCount;
+
+  #winnerCarArray;
+
+  constructor() {
+    this.#carArray = [];
+    this.#tryCount = [];
+    this.#winnerCarArray = [];
+  }
+
+  async run() {
+    await this.#getInput();
+    this.#runRace();
+    this.#printOutput();
+  }
+
+  async #getInput() {
+    const input = new Input();
+
+    await input.getUserInput();
+    this.#carArray = input.parseCars();
+    this.#tryCount = input.parseTryCount();
+  }
+
+  #runRace() {
+    const race = new Race(this.#carArray, this.#tryCount);
+
+    this.#winnerCarArray = race.run();
+  }
+
+  #printOutput() {
+    print(`${OUTPUT_MESSAGE_WINNER}${this.#winnerCarArray.join(', ')}`);
```

Comment:
```suggestion
    print(OUTPUT_MESSAGE_WINNER + this.#winnerCarArray.join(', '));
```
여기서 템플릿 리터럴을 사용하신 이유가 있으신가요?

Reply:
문자열 안에서 변수를 사용한다는 것을 명시하기 위해 사용했습니다.

여기서는 중간에 따로 들어가는 문자가 없기에, `+`를 이용해서 문자열을 잇는 것도 가독성에 있어서 오히려 더 좋은 방법인 거 같아요 !

Reply:
오.. 근데 Airbnb에 템플릿 리터럴을 권장한다는 내용이 있대요..! 

[airbnb template-literals](https://github.com/airbnb/javascript?tab=readme-ov-file#es6-template-literals)

[제 코드에 있는 리뷰](https://github.com/woowacourse-precourse/javascript-racingcar-7/pull/193/files#r1820812492) 중에 있었습니다!